docs/root/intro/arch_overview/upstream/load_balancing/panic_threshold.rst

```diff
@@ -79,8 +79,16 @@ priority.
 
 Panic mode can be disabled by setting the panic threshold to 0%.
 
-If all hosts become unhealthy normalized total health is 0%, and if the panic threshold is above 0%
-all traffic will be redirected to P=0.
+If the number of healthy hosts keeps decreasing, individual priority levels will start entering panic mode.
+Until at least one priority level is not in panic mode, load distribution is calculated as it is 
+described above. When all priority levels enter the panic mode, load calculation algorithm changes.
+In such state each priority level receives percentage of the traffic which represents the number
```

Comment:
`In this case`

`receives traffic relative to the number of hosts in that priority level`

Reply:
Done.